k8s/crd/v16/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
what is this? Why can't it be at least symlinked?

Reply:
.helmignore is automatically created when you run `helm create`... so Im not sure 🤷